q2_fmt/_stats.py

```diff
@@ -109,9 +111,17 @@ def _comp_all_pairwise(distribution, against_each=None):
             yield ((0, comp_a), (1, comp_b))
 
 
-def _compare_mannwhitneyu(group_a, group_b, p_val_approx):
-    stat, p_val = scipy.stats.mannwhitneyu(
-        group_a, group_b, method=p_val_approx, nan_policy='raise')
+def _compare_mannwhitneyu(group_a, group_b, alternative, p_val_approx):
+    if (alternative == 'two-sided' or alternative == 'greater'
+            or alternative == 'less'):
```

Comment:
This check is going to happen for each comparison, but the `alternative` argument is set only the one time, so it could be checked only once (at the top-most call-site: `mann_whitney_u`). Additionally, if you flip the conditional around, it will be a bit cleaner.